model_analyzer/result/run_config_result.py

```diff
@@ -100,12 +101,15 @@ def add_run_config_measurement(self, run_config_measurement):
             The profiled RunConfigMeasurement
         """
 
-        heapq.heappush(self._measurements, run_config_measurement)
+        inverted_rcm = RunConfigMeasurement.invert_values(
```

Comment:
These changes seem fine and mostly self-contained, but I'm still wondering why we are using heapq and jumping through all these hoops. The amount of data we are sorting is trivial. Any reason we can't just use a sorted list?

Reply:
I've switched to using bisect and remove the polarity inversion and the heap. @tgerdesnv please re-review